### PR TITLE
Made main file suck less, signup link couldn't be clicked

### PR DESCRIPTION
**Pull request by goktugyil**

### CODE REVIEW

**Comment by alavers** [comment]
Thanks! One nit, could you keep the link URL in the markdown text? eg [https://app.supportkit.io/signup](https://app.supportkit.io/signup)

File: readme.md [hunk 1]
```diff
@@ -5,10 +5,25 @@
 [![License](http://img.shields.io/cocoapods/l/SupportKit.svg)](http://supportkit.io/terms.html)
 
 # tl;dr
-    Sign up and get an app token: https://app.supportkit.io/signup
+
+
+Sign up and get an app token: [From this link](https://app.supportkit.io/signup).
```
Comment (daudich) [inline readme.md]: Thinking of [a post](http://www.smashingmagazine.com/2012/06/20/links-should-never-say-click-here/) I read recently on "Click Here" type links. Perhaps the words "Sign up" should be made clickable.

    [Sign up](https://app.supportkit.io/signup) and get an app token.

OR

    [Sign up](https://app.supportkit.io/signup) with SupportKit and get an app token.

**Comment by daudich** [comment]
:+1:

**Comment by goktugyil** [comment]
updated link, daudich thanks for the post, that also looks good

**Comment by jpjoyal** [comment]
Thank you!!

**Pull request merged**